libs/client/ui/pages/src/lib/profile/profile.tsx

```diff
@@ -219,189 +223,162 @@ export const Profile: React.FC<ProfileProps> = inject(userStore.storeKey)(
               },
             ]}
           />
-          {isGuest && (
-            <div className="guest-lock-modal">
-              <IonGrid>
-                <IonRow>
-                  <IonIcon
-                    className="lock-icon"
-                    icon={lockClosed}
-                    color="primary"
-                  />
-                </IonRow>
-                <IonRow className="info">
-                  <IonText color="primary">
-                    Erstelle ein Konto, um auf dein Profil zugreifen zu können.
-                  </IonText>
-                </IonRow>
-                <IonRow>
-                  <IonButton
-                    expand="block"
-                    mode="ios"
-                    fill="solid"
-                    onClick={() =>
-                      handleLogout('/register', userStore.user?.role)
-                    }
-                  >
-                    {isLoggingOut ? (
-                      <IonLabel>
-                        <IonSpinner name="crescent" />
-                      </IonLabel>
-                    ) : (
-                      'Konto erstellen'
-                    )}
-                  </IonButton>
-                </IonRow>
-              </IonGrid>
-            </div>
-          )}
           <IonGrid>
```

Comment:
Kansch des Lock  bitte uf glicher höhe wie IonToast macha. Sunsch isch es vo da größe vom Grid abhängig

Reply:
Jooaa des isch abiz blöd weil des grid alles umfassen sött. Sus crashts glob für da igloggte User.

Reply:
? Wie des. Des ion-toast isch jo o außerhalb und des schloss wird für igloggte User garnet geladen und i hobs jo in nam pull-request davor jo grad so geändert

Reply:
jo aba des schloss isch im selba component wia da blur. des hobi jo usglagert in da `GuestLock`